### ISSUE TITLE
The learning rate of Lenet

### ISSUE THREAD

**Issue by wjp97** [Ksuryateja/pytorch-cifar10#1]
What's the learning rate of lenet? 
And are there any changes of hyper-parameters for lenet?

**Comment by Ksuryateja** [comment]
Learning rate is 0.001, it is mentioned in the adam optimizer in cifar10.py

Architecture is the same as the original LeNet but max pooling instead of Average pooling and Relu instead of tanh activation was used.
This is the original architecture.
![LeNEt_Summary_Table](https://user-images.githubusercontent.com/37737257/65680786-4b432d80-e075-11e9-9c4e-3c4e4536d85c.jpg)
 You can find the architecture of LeNet used in models
